presubmit.py

```diff
@@ -241,14 +241,12 @@ def pytype(paths: List[Path]) -> bool:
     """Run pytype on |path| if it is a python file. Return False if it fails
     type checking."""
     paths = [path for path in paths if is_python(path)]
-    if not paths:
```

Comment:
Is this (and the print below) an intentional change?

Reply:
Yeah. We don't need this early bail out. The print wasn't intended though.